lib/cmd/license-file-handler.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const fs = require('fs');
+
+class LicenseFileHandler
+{
+  static getFile(filename)
+  {
+    return new Promise((resolve,reject) => {
```

Comment:
Node.js performance is not good to synchronous process, but as it is a ".bat"/."sh" there isn't a reason to worry and use sync can be a handful thing

You don't need to use promise since it is not async. Why have you prefered to use promise?